uefi/src/table/boot.rs

```diff
@@ -1139,6 +1139,30 @@ impl BootServices {
         assert_eq!((self.stall)(time), Status::SUCCESS);
     }
 
+    /// Adds, updates, or removes a configuration table entry
+    /// from the EFI System Table.
+    ///
+    /// # Safety
+    ///
+    /// This relies on `table_ptr` being allocated in the
+    /// pool of type `EfiRuntimeServicesData`.
```

Comment:
Shouldn't this point to uefi-rs types instead of the C names.

Also, strictly speaking, other types can be okay to use too (kernel uses `EfiACPIReclaimMemory` for rng seeds for example).